client/src/components/categorical/category/index.tsx

```diff
@@ -641,13 +643,48 @@ const CategoryValueList = React.memo(
     colorData,
     colorTable,
     colorMode,
+    isCellGuideCxg,
   }: CategoryValueListProps) => {
-    const tuples = [...categorySummary.categoryValueIndices];
+    let tuples = [...categorySummary.categoryValueIndices];
+    if (isCellGuideCxg) {
+      tuples = tuples.filter(
+        ([, index]) => categorySummary.categoryValueCounts[index] > 0
+      );
 
-    /*
-    Render the value list.  If this is a user annotation, we use a flipper
-    animation, if read-only, we don't bother and save a few bits of perf.
-    */
+      if (
```

Comment:
like the code block below, maybe also add a comment to explain what this code block does, so other readers understand the intention of this code